shared-objects/servers.js

```diff
@@ -24,6 +34,10 @@ const config = {
 		PORT: process.env.IT_MANAGEMENT_SERVER_PORT || '3333',
 		get URL() {
 			return `${config.MANAGEMENT_SERVER.PROTOCOLL}://${config.MANAGEMENT_SERVER.HOST}:${config.MANAGEMENT_SERVER.PORT}/api`;
+			/**
+			* needed when running the tests locally
+			* return 'http://mgmt-svc.${instance}-${localBranchName}.svc.cluster.local:3333/api/management/database/seed';
```

Comment:
We might need to talk about this one. Uzair tried this, when running tests locally, but got an error. Not sure what we missed.